internal/buffer/settings.go

```diff
@@ -49,6 +50,10 @@ func (b *Buffer) SetOptionNative(option string, nativeValue interface{}) error {
 			b.UpdateRules()
 		}
 	} else if option == "encoding" {
+		_, err := htmlindex.Get(b.Settings["encoding"].(string))
+		if err != nil {
+			return err
```

Comment:
We don't really check this return value (thus don't report the error to the user) in `reload`...

Reply:
Holy, yep...we didn't consider it so far. :+1: